roles/installer/tasks/59_power_off_cluster_servers.yml

```
@@ -15,10 +15,37 @@
         ( item not in groups['dell_hosts_redfish'] | default([]) )
           and
         ( item not in groups['hp_hosts_redfish'] | default([]) )
-      with_items:
+      loop:
         - "{{ groups.masters }}"
         - "{{ groups.workers | default([]) }}"
```

The change to loop is causing it to fail, this will get it to work
```suggestion
      loop: "{{ groups.masters + groups.workers | default([]) }}"
```